rust-lib/Cargo.toml

```
@@ -16,18 +16,12 @@ openssl = { version = "^0.10", features = ["vendored"] }
 
 [dependencies]
 log = "0.4"
-rgb_node = { git = "https://github.com/LNP-BP/rgb-node", branch = "master" }
+rgb_node = { git = "https://github.com/LNP-BP/rgb-node", version = "~0.1.0" }
```

We do not need git here at all
```suggestion
rgb_node = "~0.1.0"
```

---

Sorry, I misunderstood, fixed it